pages/hackathon/solanathon.tsx

```diff
@@ -196,7 +196,7 @@ const SolanaHackathon: ThirdwebNextPage = () => {
           mt="-150px"
         />
         <Judges />
-        <HackathonFooter />
+            <HackathonFooter />
```

Comment:
remove the extra whitespace here please :)

Reply:
oops sorry about that,
used prettier on all the code it should be fine now!